styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesGCPServiceAccountSecretManager.java

```diff
@@ -118,30 +118,36 @@ String ensureServiceAccountKeySecret(String workflowId,
   private String getOrCreateServiceAccountKeySecret(
```

Comment:
A refactoring for this method? It doesn't fit into one page for my browser any more. ;)